app/src/main/res/layout/activity_main.xml

```diff
@@ -18,7 +18,8 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:stepCount="5"
             app:currentCount="4"
-            app:drawLabels="true"
+            app:drawLabels="false"
+            app:drawCount="true"
```

Comment:
Also can you check if status data is longer like 
"ABCD" does it exceed the circle or looks okay.
Actually it does, so we might have to handle it
<img width="222" alt="Screen Shot 2019-11-17 at 5 29 01 PM" src="https://user-images.githubusercontent.com/28902512/69003035-c8956b00-095f-11ea-8e8a-f02a18f7eddd.png">